app/src/main/res/values-de/strings.xml

```diff
@@ -84,11 +84,11 @@
     <string name="decline_button_cnt_des">Transfer ablehnen</string>
     <string name="accept_button_cnt_des">Transfer annehmen</string>
     <string name="stop_button_cnt_des">Transfer anhalten</string>
-    <string name="retry_button_cnt_des">Transfer erneut versuchen</string>
+    <string name="retry_button_cnt_des">Transfer erneut starten</string>
```

Comment:
uDEV2019 wrote, --> sphenotux comments
Line 87 says "retry" and this means "Versuch" and not "Start". --> Yes, "retry" means "Versuch" - but if I read the word "versuchen" - try - in German, it wakes some kind of distrust, as it means: Give it a try, but you may fail. IMHO a bit weaker in English using "retry". 
For me, "starten" means - we just **do** it. --> In addition to line 49, closest to English is "versuchen".